clover_web.rb

```diff
@@ -294,6 +294,10 @@ def redirect_back_with_inputs
     request.redirect env["HTTP_REFERER"]
   end
 
+  def has_project_permission(actions)
+    (@project_permissions & Authorization.extend_actions(actions)).any?
```

Comment:
bitwise and? that seems...weird. Intentional?

Reply:
It returns intersection of two arrays. Do we have any other way to get intersection of two arrays in Ruby?

Reply:
There is an `Array#intersection`. Wordy yes, but also forms a redundant type check of sorts.

Reply:
Thank you for the recommendation. I'm not sure why ChatGPT suggested this one, but it does appear more readable. I've made the change.